src/pilot/inspect.patch

```diff
@@ -0,0 +1,70 @@
+--- inspect.py	2019-03-18 04:26:06.117996201 -0400
++++ inspect_latest.py	2019-03-18 04:36:58.007653944 -0400
+@@ -119,10 +119,17 @@
+                       {'node_uuid': node.uuid, 'error': exc})
+             raise exception.HardwareInspectionFailure(error=exc)
+ 
++        pxe_dev_nics = self._get_pxe_dev_nics(client, nics, node)
++        if pxe_dev_nics is None:
++            LOG.warning(_LW('No PXE enabled NIC was found for node '
++                        '%(node_uuid)s.'), {'node_uuid': node.uuid})
++        
+         for nic in nics:
+             try:
+                 port = objects.Port(task.context, address=nic.mac,
+-                                    node_id=node.id)
++                                    node_id=node.id,
++                                    pxe_enabled=(nic.id in pxe_dev_nics))
++                
```

Comment:
nit: This newline is not in the Queens version that contains PXE enabled. Instead, it is below, between lines 19 and 20.

Reply:
removed this new line in latest commit and added below as you mentioned.